HW3/HW3.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+   <meta charset="UTF-8">
+   <meta http-equiv="X-UA-Compatible" content="IE=edge">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <title>HW3</title>
+   <link rel="stylesheet" href="./style/style.css">
+   <link rel="preconnect" href="https://fonts.googleapis.com">
+   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+   <link href="https://fonts.googleapis.com/css2?family=Open+Sans&display=swap" rel="stylesheet">
+
+
+</head>
+
+<body>
+   <div class="header">
+      <div class="header__logo">
+         <img src="./images/AXIT-1.svg" alt="AXIT">
+         <img src="./images/AXIT.svg" alt="AXIT">
+      </div>
+      <ul class="header__nav-list">
+         <li><a href="link1.html">Features</a></li>
+         <li><a href="link2.html">About</a></li>
+         <li><a href="link3.html">Pricing</a></li>
+         <li><a href="link4.html">Reviews</a></li>
+         <li><a href="link5.html">Contacts</a></li>
+      </ul>
+   </div>
+   <div class="main-block">
+      <div class="main-block__content">
+         <img class="main-block__logo" src="./images/logo.png" alt="Logo">
+         <div>
+            <p class="main-block__heading">Modern axure template <br> for beutiful <br> prototypes </p>
+
+            <hr class="main-block__line">
+
+            <p class="main-block__text">Lorem ipsum dolor sit amet,<br> consectetur adipisicing elit</p>
+            <button class="main-block__button">Download</button>
+         </div>
+         <div class="main-block__form">
+            <p class="main-block__form__heading">try your <span class="main-block__form__red"> Free </span> trial today
```

Comment:
when you write class like main-block__form__heading
you mean that block__element_modification
modification means that something was change
you may write like main-block__form-heading